src/acquisition/covidcast/database.py

```diff
@@ -560,68 +560,69 @@ def get_covidcast_meta(self):
 
     sql = 'SELECT `source`, `signal` FROM covidcast WHERE NOT `is_wip` GROUP BY `source`, `signal` ORDER BY `source` ASC, `signal` ASC;'
     self._cursor.execute(sql)
-    for source, signal in  [ss for ss in self._cursor]: #NOTE: this obfuscation protects the integrity of the cursor; using the cursor as a generator will cause contention w/ subsequent queries
+    signals = [ss for ss in self._cursor] #NOTE: this obfuscation protects the integrity of the cursor; using the cursor as a generator will cause contention w/ subsequent queries
+    for source, signal in signals:
+
+      # calculate the min issues per combination
+      sql = '''
+      SELECT
+        `time_type`,
+        `geo_type`,
+        MIN(`issue`) as `min_issue`
+      FROM
+        `covidcast`
+      WHERE
+        `source` = %s AND
+        `signal` = %s
+      GROUP BY
+        `time_type`,
+        `geo_type`
+      '''
+      self._cursor.execute(sql, (source, signal))
+      min_issue_lookup = {f'{x[0]}:{x[1]}': x[2] for x in self._cursor}
```

Comment:
you can use a tuple as the key for a dict.  it probably wouldnt make much (if any) of a performance difference here, but it would make for a shorter comprehension.  its really just a style judgement call either way ¯\_(ツ)_/¯

Reply:
done